spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/repository/it/SpannerRepositoryIntegrationTests.java

```diff
@@ -488,9 +480,6 @@ void queryMethodsTest_EagerFetch() {
         .hasValueSatisfying(t -> assertThat(t.getSymbol()).isEqualTo(aTrade.getSymbol()))
         .hasValueSatisfying(
             t -> assertThat(t.getSubTrades()).hasSize(aTrade.getSubTrades().size()));
-    Mockito.verify(spannerTemplate, Mockito.times(1)).executeQuery(any(Statement.class), any());
-    Mockito.verify(spannerTemplate, Mockito.times(1))
-        .query(eq(Trade.class), any(Statement.class), any(SpannerQueryOptions.class));
```

Comment:
These lines verify whether the `executeQuery` and `query` are called with the given parameter types which should be implicitly checked in the lines above.  From the style guide, "Verification should only be done if calling that exact method is crucial to the correct functioning of the code" which makes me think that we are over-asserting.  Looking for a second opinion on this?

Reply:
Updating this comment for posterity: As discussed offline, verified that this check is redundant as the # invocations for `executeQuery` are covered through unit tests under `SqlSpannerQueryTests`.